cli/popper/utils.py

```diff
@@ -98,16 +111,28 @@ def is_popperized():
     return os.path.isfile(config_filename)
 
 
-def update_config(name, stages, envs, relative_path):
-    """Updates the configuration for a pipeline"""
+def update_config(name, stages='', envs='', vars=[], relative_path=''):
+    """Updates the configuration for a pipeline."""
 
     if name == 'paper':
         stages = 'build'
 
     config = read_config()
+    if config['pipelines'].get(name, None):
+        if not stages:
+            stages = ','.join(config[name]['stages'])
+        if not envs:
+            envs = ','.join(config[name]['envs'])
+        if not relative_path:
+            relative_path = config[name]['path']
+
+    if name == 'paper':
```

Comment:
Can this control statement be removed? 
Since it's already present in lines 117-118